jekyll/_cci2/overview.md

```diff
@@ -0,0 +1,128 @@
+
+
+| Source                      | Ports                   | Use                    |
+|-----------------------------|-------------------------|------------------------|
+| End Users                   | 80, 443                 | HTTP/HTTPS Traffic     |
+| Administrators              | 22                      | SSH                    |
+| Administrators              | 8800                    | Admin Console          |
+| Builder Boxes               | all traffic / all ports | Internal Communication |
+| GitHub (Enterprise or .com) | 80, 443                 | Incoming Webhooks      |
+{: class="table table-striped"}
+
+### Builders
+The Builder instances run without storing state, enabling you to increase or decrease containers as needed. To ensure that there are enough Builder machines running to handle all of the builds, track the queued builds and increase the Builder instance machines as needed to balance the load.
+
+Each machine on which the Builders are installed reserves two CPUs and 4GB of memory for coordinating builds. The remaining processors and memory create the containers. Larger machines are able to run more containers and are limited by the number of available cores after two are reserved for coordination. The following table describes the ports used on the Builder instances:
+
+
+| Source                           | Ports                   | Use                                                            |
+|----------------------------------|-------------------------|----------------------------------------------------------------|
+| End Users                        | 64535-65535             | [SSH into builds feature](https://circleci.com/docs/2.0/ssh-access-jobs/) |
+| Administrators                   | 80, 443                 | CircleCI API Access (graceful shutdown, etc)                   |
+| Administrators                   | 22                      | SSH                                                            |
+| Services Box                     | all traffic / all ports | Internal Communication                                         |
+| Builder Boxes (including itself) | all traffic / all ports | Internal Communication                                         |
+{: class="table table-striped"}
+
+### GitHub
+CircleCI uses GitHub or GitHub Enterprise credentials for
+authentication which, in turn, may use LDAP, SAML, or SSH for access. That is, CircleCI will inherit the authentication  supported by your central SSO infrastructure. The following table describes the ports used on machines running GitHub to communicate with the Services and Builder instances.
+
+
+| Source        | Ports   | Use          |
+|---------------|---------|--------------|
+| Services   | 22      | Git Access   |
+| Services   | 80, 443 | API Access   |
```

Comment:
Consider "or" instead of the comma on this and two lines down to show that if you're using SSl you don't need to keep port 80 open.

Reply:
thanks added